### PR TITLE
bugfix [P6M-615]: Add voyageai to v2 embedders

### PR DESCRIPTION
**Pull request by potter-potter**
VoyageAI added to V2 embedders and tested with a new login.

Not sure if we want to add the login to the CI tests. The billing is sort of unknown.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -82,8 +83,6 @@
...
 * **Textual content retrieved from a URL with gzip transport compression now partitions correctly.** Resolves a bug where a textual file-type (such as Markdown) retrieved by passing a URL to `partition()` would raise when `gzip` compression was used for transport by the server.
-* **A DOCX, PPTX, or XLSX content-type asserted on partition is confirmed or fixed.** Resolves a bug where calling `partition()` with a swapped MS-Office `content_type` would cause the file-type to be misidentified. A DOCX, PPTX, or XLSX MIME-type received by `partition()` is now checked for accuracy and corrected if the file is for a different MS-Office 2007+ type.
-* **DOC, PPT, XLS, and MSG files are now auto-detected correctly.** Resolves a bug where DOC, PPT, and XLS files were auto-detected as MSG files under certain circumstances.
```
Comment (ryannikolaidis) [inline CHANGELOG.md:86]: why was this removed?

**Review by ryannikolaidis — CHANGES_REQUESTED** [review]
good with the change, but qq on removed changelog line

File: unstructured/ingest/v2/processes/embedder.py [hunk 2]
```diff
@@ -58,6 +58,13 @@ def get_embedder(self) -> BaseEmbeddingEncoder:
             )
 
             return VertexAIEmbeddingEncoder(config=VertexAIEmbeddingConfig(**kwargs))
+        elif self.embedding_provider == "langchain-voyageai":
+            from unstructured.embed.voyageai import (
+                VoyageAIEmbeddingConfig,
+                VoyageAIEmbeddingEncoder,
+            )
+
+            return VoyageAIEmbeddingEncoder(config=VoyageAIEmbeddingConfig(**kwargs))
```
Comment (ryannikolaidis) [inline unstructured/ingest/v2/processes/embedder.py:67]: this makes sense and follows convention. aside: there _has_ to be a better way to do this than an ever-growing if-else chain.. :/

**Comment by potter-potter** [comment]
@ryannikolaidis Sorry to lead you to look at this. Voyageai v2 was already added to unstructured-ingest.

https://github.com/Unstructured-IO/unstructured-ingest/blob/78af5635497d6fa24eae37dae7ac827416437c13/unstructured_ingest/v2/processes/embedder.py#L88

**Comment by ryannikolaidis** [comment]
> @ryannikolaidis Sorry to lead you to look at this. Voyageai v2 was already added to unstructured-ingest.
> 
> https://github.com/Unstructured-IO/unstructured-ingest/blob/78af5635497d6fa24eae37dae7ac827416437c13/unstructured_ingest/v2/processes/embedder.py#L88

eek. that pr doesn't even mention this change :/
